environment.yml

```diff
@@ -21,4 +21,5 @@ dependencies:
   - pip
   - pip:
       - pyscaffold
-      - sphinx_rtd_theme # to build docs locally
+      - sphinx_rtd_theme  # to build docs locally
+      - -e .[testing]  # install git checkout of PyScaffold in editable mode
```

Comment:
Following up our discussion yesterday I believe it would be better to leave the extra `[testing]` out, right? Otherwise pip will install pytest and pytest-cov, which could break conda compatibility...

I suppose `- -e .` would be enough, and then we leave all of the testing stuff for the conda-patched version of tox... After all the user do not need to have pytest directly installed in the dev environment (no playing around with pytest on the console or training/data processing scripts) if we assume all the tests are managed by tox (which is self contained).

Reply:
Good point. To my understanding, conda first installs all own packages and in the last step installs the pip packages which are then only installed if not already satisfied by conda packages. Otherwise also the direct non-testing dependencies of the package itself might be problematic (assuming for instance you write a package that depends on virtualenv directly). So I think we are safe here but I'll remove it anyway since it also doesn't help :-)